_config.yml

```diff
@@ -416,4 +416,21 @@ contributors:
       site: https://thecodeangle.com/
       twitter: https://twitter.com/home
       github: https://github.com/desoga10
-      linkedin: https://www.linkedin.com/in/desoga/
\ No newline at end of file
+      linkedin: https://www.linkedin.com/in/desoga/
+  "Saurabh Ghatnekar":
+    slug: saurabh-ghatnekar
+    desc: Saurabh is a Full Stack Developer. He is passionate about learning and sharing knowledge. He is a contributor to the React and JS community.
+    specs: [javascript, react, nodejs, mongodb, express, AWS]
+    image: /img/contributors/saurabh-ghatnekar.png

```

Comment:
Please add your image to the repo. You can put it here
**themes/my-theme/source/img/contributors**